src/fastaudio/core/signal.py

```diff
@@ -50,32 +48,6 @@ def tar_extract_at_filename(fname, dest):
     tarfile.open(fname, "r:gz").extractall(dest)
 
 
-# fix to preserve metadata for subclass tensor in serialization
-# src: https://github.com/fastai/fastai/pull/3383
-# TODO: remove this when #3383 lands and a new fastai version is created
```

Comment:
This PR went into place so I removed these because `self.storage` was causing issues